core/src/main/java/com/amplifyframework/auth/AuthProvider.java

```diff
@@ -22,10 +22,10 @@
  * A factory for some preset third party auth providers and the ability to create a new, custom one.
  */
 public class AuthProvider {
-    private static final String AMAZON = "www.amazon.com";
-    private static final String FACEBOOK = "graph.facebook.com";
-    private static final String GOOGLE = "accounts.google.com";
-    private static final String TWITTER = "api.twitter.com";
+    private static final String AMAZON = "amazon";
+    private static final String FACEBOOK = "facebook";
+    private static final String GOOGLE = "google";
+    private static final String APPLE = "apple";
```

Comment:
Does Android SDK currently support sign-in with apple?

Reply:
This is for passing to HostedUI which based on the Cognito documentation I read does seem to support it.